### PR TITLE
github: Initial templates

### PR DESCRIPTION
**Pull request by elms**
* SECURITY.md
 * PR template
 * Issue templates (bug and general)

### CODE REVIEW

**Comment by elms** [comment]
# Issue Templates
gh_issues

## Bug
gh-bug

## General issue
gh-general

## Security Vulnerability
gh-security

# PR template
gh-pr-template

**Review by dgarske — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    id: version
+    attributes:
+      label: Version
+      description: What version were you using?
+    validations:
+      required: true
+  - type: textarea
+    id: details
+    attributes:
+      label: description
```
Comment (dgarske) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:29]: For consistency please make this `Description`.

**Review by dgarske — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (elms) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: Done

**Review by dgarske — APPROVED** [review]
This is awesome! Thanks Jeff

**Pull request merged**